datarepo-client/build.gradle

```diff
@@ -40,64 +25,43 @@ repositories {
 }
 
 dependencies {
-    ext {
-        commonsLang3 = "3.9"
-        googleApiServicesOauth2 = "v2-rev20190313-1.30.1"
-        googleClient = "1.30.7"
-        googleCloudStorage = "1.103.1"
-        googleOauthClientJetty = "1.30.5"
-        jackson = "2.10.2"
-        jersey = "2.30.1"
-        junit = "4.13"
-        swagger = "1.5.22"
-        hamcrest = "2.2"
-    }
-
-    compile group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: "${jackson}"
-    compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: "${jackson}"
-    compile group: 'io.swagger', name: 'swagger-annotations', version: "${swagger}"
-    compile group: 'org.glassfish.jersey.core', name: 'jersey-client', version: "${jersey}"
-    compile group: 'org.glassfish.jersey.media', name: 'jersey-media-json-jackson', version: "${jersey}"
-    compile group: 'org.glassfish.jersey.media', name: 'jersey-media-multipart', version: "${jersey}"
-    compile "org.glassfish.jersey.inject:jersey-hk2:${jersey}"
-    compile group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: "${jackson}"
-
-    testCompile group: 'junit', name: 'junit', version: '4.12'
-    testCompile group: 'org.hamcrest', name: 'hamcrest', version: '2.1'
+    // Swagger deps, versions controlled by dependencyManagement in root project
+    implementation group: "org.glassfish.jersey.core", name: "jersey-client"
+    implementation group: "org.glassfish.jersey.media", name: "jersey-media-json-jackson"
+    implementation group: "org.glassfish.jersey.media", name: "jersey-media-multipart"
+    implementation group: "com.fasterxml.jackson.datatype", name: "jackson-datatype-jsr310"
+    implementation group: "io.swagger.core.v3", name: "swagger-annotations"
+    swaggerCodegen group: "io.swagger.codegen.v3", name: "swagger-codegen-cli"
```

Comment:
I am not sure exactly how to keep the original dependencies while using the new syntax, but we will otherwise have a conflict with some of the `unauthenticatedApi` generated code.